Orange/widgets/utils/owlearnerwidget.py

```diff
@@ -123,7 +123,7 @@ def __init__(self):
         self.preprocessors = None
         self.outdated_settings = False
         self.setup_layout()
-        QTimer.singleShot(0, self.apply)
+        QTimer.singleShot(0, self.update_learner)
```

Comment:
Wouldn't it be safer to call `self.unconditional_apply` here? It doesn't change anything for the base class since `update_model` doesn't do anything without the data, but derived classes can add some code to the `apply` method or change `update_model` to do something without data.